apps/aecore/src/aec_pow_cuckoo.erl

```diff
@@ -269,9 +266,6 @@ get_executable(#miner_config{executable = Executable}) ->
 get_extra_args(#miner_config{extra_args = ExtraArgs}) ->
     ExtraArgs.
 
-get_nice(#miner_config{nice = Nice}) ->
-    Nice.
```

Comment:
So nice is no longer supported?

Reply:
Not for now, I didn't bother re-implementing it since its usefulness kind of diminishes with the increased usage of pool mining.

Reply:
So maybe we should either remove `nice` property from the config schema, or at least mark it as deprecated there. I mean `epoch_config_schema.json`, and removal from all sample configs from all tests?

Reply:
Good point. Removing it would make the release incompatible with previous configurations, so I'd prefer marking it deprecated. Will do that.

Reply:
I've made the requested changes.